android/src/main/java/com/swmansion/rnscreens/FragmentHolder.kt

```diff
@@ -0,0 +1,7 @@
+package com.swmansion.rnscreens
+
+import androidx.fragment.app.Fragment
+
+interface FragmentHolder {
+    val fragment: Fragment
+}
```

Comment:
Decided to split this property to separate interface as this is the key idea behind this PR.

Also in the future I plan to make `ScreenFragmentWrapper` interfaces much more granular (split those into number of fine-grained interfaces.